docs/Configuring-Pantheon/Consensus-Mechanisms.md

```diff
@@ -0,0 +1,52 @@
+# Comparing Proof of Authority Consensus Protocols  
+
+Pantheon implements the Clique and IBFT 2.0 Proof of Authority consensus protocols. Proof of Authority 
+consesus protocols are used when participants are known to each other and there is a level of trust between them. 
+For example, in a permissioned consortium network. 
+
+Proof of Authority consensus protocols are faster and allows much greater throughput of transactions 
+than the Ethash Proof of Work consensus protocol used on the Ethereum MainNet. 
+
+In Clique and IBFT 2.0, a group of nodes in the network act as signers (Clique) or validators (IBFT 2.0). These nodes propose, validate, 
+and add blocks to the blockchain. Nodes are added to or removed from the signer/validator pool by the existing group of nodes voting. 
+
+!!! note
+     For the rest of this page, the term validator is used to refer to signers and validators. 
+
+## Properties 
+   
+Properties to consider when comparing Clique and IBFT 2.0 are: 
+
+* Immediate finality 
+* Liveness 
+* Speed 
+* Minimum number of validators 
+
+### Immediate Finality 
+
+IBFT 2.0 has immediate finality. When using IBFT 2.0 there are no forks and all valid blocks are included in the main chain.
+
+Clique does not have immediate finality. Implementations using Clique must be aware of forks and chain reorganizations occurring. 
```

Comment:
chain reorganizations occurring -> chain reorganizations in progress

Reply:
I'm leaving this -> you need to be aware of chain reorganizations after they have occurred as well as when they are in progress. eg, when you're accessing logs

Reply:
Makes sense.